presto-hive/src/main/java/io/prestosql/plugin/hive/HdfsConfigurationInitializer.java

```diff
@@ -50,7 +50,7 @@
 import static org.apache.hadoop.hive.conf.HiveConf.ConfVars.COMPRESSRESULT;
 import static org.apache.hadoop.io.SequenceFile.CompressionType.BLOCK;
 
-public class HdfsConfigurationUpdater
+public class HdfsConfigurationInitializer
```

Comment:
- I don't get this rename, I would keep the old name.
- why this does not implement `DynamicConfigurationProvider` (`ConfigurationUpdater`)?

Reply:
See my previous comment. This class is to initialize the newly created configuration. It's not dependent on the target URI or context.

Reply:
Then method `updateConfiguration` needs to be renamed as well.